pkgs/by-name/ca/cargo-autoinherit/package.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-autoinherit";
+  version = "0.1.4";
+
+  src = fetchFromGitHub {
+    owner = "mainmatter";
+    repo = pname;
```

Comment:
```suggestion
    repo = "cargo-autoinherit";
```

Reply:
Why?

Reply:
https://github.com/NixOS/nixpkgs/issues/277994

There is no necessary connection between the two, and it is useless to establish this relationship between them.